viewer/src/main/webapp/viewer-html/common/viewercontroller/ViewerController.js

```diff
@@ -2379,5 +2381,21 @@ Ext.define("viewer.viewercontroller.ViewerController", {
             }
         }
         delete this.components;
+    },
+
+    checkAppLayersForServiceLayer: function () {
+        for (var appLayer in this.app.appLayers) {
+            var found = false;
+            for (var serviceId in this.app.services){
+                if (this.app.appLayers[appLayer].serviceId === this.app.services[serviceId].id) {
+                    found = true;
+                    break;
+                }
+            }
+            if (!found) {
+                console.log("No service layer found for: " + this.app.appLayers[appLayer].layerName)
```

Comment:
Ik denk dat deze `console.log(..)` regel weg kan, 
- niemand ziet dat
- levert problemen op in oude browsers (die in oude versies nog wel worden ondersteund) waar de backports voor zijn gelabeld

niet @mtoonen ?

Reply:
Weg doen dan maar?